configure.ac

```
@@ -119,6 +119,11 @@ AC_DEFINE_UNQUOTED([ENABLE_PYTHON],
 	[$(test "$enable_python" = yes && echo 1 || echo 0)],
 	[Python bindings build flag.])
 
+AC_CHECK_TOOL(GPERF, gperf)
+if test -z "$GPERF"; then
+	AC_MSG_ERROR(please install gperf)
```

This is a nit, but since we still need some other changes, how about the following (use square brackets) for the sake of consistency with the other `AC_MSG_ERROR()` call in configure.ac.
```suggestion
	AC_MSG_ERROR([please install gperf])
```